docker-entrypoint

```diff
@@ -94,6 +114,9 @@ if [ "$1" = "autoheal" ] && [ -e "$DOCKER_SOCK" ];then
         if ! restart_container "$CONTAINER_ID" "$TIMEOUT"
         then
           echo "$DATE Restarting container $CONTAINER_SHORT_ID failed" >&2
+          notify_webhook "Container ${CONTAINER_NAME:1} (${CONTAINER_SHORT_ID}) found to be unhealthy. Failed to restart the container!"
```

Comment:
should webhooks be as a background process, as healer shouldn't wait for webhook to complete  to restart other possibly critical containers unhealthy in loop?

Reply:
I like that idea!

Reply:
I added a new commit where the webhook curl commands run as background processes.

Reply:
would it be better to make function call with background rather than in just curl statement in definition?

Reply:
imho running just the curl command as a background process is fine as it is the only command that might block the parent process (healer). it also makes it less likely to miss the background process operator if it is used somewhere else in the script. but I am open to other opinions.

Reply:
I was more leaning to all statements blocking/nonblocking in notify_webhook are sorta in the way of next heal.